src/components/Navbar.tsx

```diff
@@ -235,6 +251,17 @@ export const Navbar = () => {
                     <aside className="flex flex-col items-start justify-between h-[90%]">
                         <ul className="flex flex-col items-start justify-start gap-4 w-full">
                             {currentUser && renderNavItems(false)}
+                            <a
+                                href="https://maps.app.goo.gl/Fxic5XJBzZjHP4Yt5"
+                                target="_blank"
+                                rel="noopener noreferrer"
+                                className="w-full"
+                            >
+                                <li className="p-4 hover:bg-slate-100 duration-300 transition-colors rounded-md w-full hover:text-black cursor-pointer flex items-center justify-start gap-2">
+                                    <FiMapPin size={32} />
+                                    Location
+                                </li>
+                            </a>
```

Comment:
Not sure if we want to show the location unless the user is `mentor/volunteer/speaker/sponsor` or if its `hacker` then we only show when they have been accepted and rsvp'd

Reply:
The mobile navbar also needs updating.

Reply:
The hacker condition is missing `currentUser.type === 'hacker' && currentUser.rsvpVerified`